D1673/P1673.md

```diff
@@ -8550,11 +8531,15 @@ void triangular_matrix_vector_solve(
   BinaryDivideOp divide);
 ```
 
-[1]{.pnum} *Effects:* Computes $x'$ such that $b = A x'$,
+[5]{.pnum} This function performs a triangular solve taking into account the `Triangle` and `DiagonalStorage` parameters [linalg.general].
```

Comment:
Should we consider distinguishing this from the in-place triangular solve below?

```suggestion
[5]{.pnum} This function performs a not-in-place triangular solve taking into account the `Triangle` and `DiagonalStorage` parameters [linalg.general].
```

Reply:
I think it sounds awkward and its clear from the other function that that is in-place.

Reply:
I'd prefer to distinguish this somehow from the other case.  Would you happen to have a suggestion for another phrasing?